client/components/redux-forms/redux-form-radio/index.jsx

```diff
@@ -1,43 +1,24 @@
+/** @format */
 /**
  * External dependencies
  */
-import React, { Component, PropTypes } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import { Field } from 'redux-form';
-import { omit } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import FormRadio from 'components/forms/form-radio';
 
-class ReduxFormRadio extends Component {
-	static propTypes = {
-		name: PropTypes.string,
-		value: PropTypes.string,
-	};
+// eslint-disable-next-line no-unused-vars
+const RenderRadio = ( { input, meta, type, ...props } ) => <FormRadio { ...input } { ...props } />;
```

Comment:
Components should be nouns, not verbs (`Render...`) -- should we just call it `Radio` now?